cmd/app/link_test.go

```diff
@@ -95,15 +95,15 @@ func Test_Apps_Link(t *testing.T) {
 				).Return(api.GetAppStatusResult{}, nil)
 			},
 			CmdArgs: []string{},
-			ExpectedAsserts: func(t *testing.T, cm *shared.ClientsMock) {
+			ExpectedAsserts: func(t *testing.T, ctx context.Context, cm *shared.ClientsMock) {
 				expectedApp := types.App{
 					AppID:        mockLinkAppID1,
 					TeamDomain:   mockLinkSlackAuth1.TeamDomain,
 					TeamID:       mockLinkSlackAuth1.TeamID,
 					EnterpriseID: mockLinkSlackAuth1.EnterpriseID,
 				}
 				actualApp, err := cm.AppClient.GetDeployed(
-					context.Background(),
+					ctx,
```

Comment:
note: The main motivation of this PR is to replace `context.Background()` references with the mocked context `ctx`. Here's one example.

Reply:
Replacing these instances with the shared mock `ctx` IMO is a huge win for all of our tests 😭